lib/src/kbasesearchengine/events/handler/WorkspaceEventHandler.java

```diff
@@ -322,6 +323,42 @@ private void checkStorageCode(final String storageCode) {
         };
     }
 
+    public Tuple9<Long, String, String, String, Long, String, String,
+            String,Map<String,String>> getWorkspaceInfo(
+            final long wsId)
+            throws IOException, JsonClientException {
+
+        final Map<String, Object> command = new HashMap<>();
+        command.put("command", "getWorkspaceInfo");
+        command.put("params", new WorkspaceIdentity()
+                .withId(wsId));
+
+        final Tuple9<Long, String, String, String, Long, String, String,
+                String,Map<String,String>> wsInfo;
+
+        wsInfo = ws.getClient().administer(new UObject(command))
+                .asClassInstance(WS_INFO_TYPEREF);
+
+        return wsInfo;
+    }
+
+//    public Map<String, String> getWorkspaceInfo(
```

Comment:
delete commented out code

Reply:
Done. Sorry, should have taken care of that before.